### DISCUSSION TITLE
Using geopandas to explore a geojson LineString

### DISCUSSION THREAD

**Discussion by eric-g-97477** [geopandas/geopandas#2768] [Q&A] [answered]
I have the following python code:

```
import geopandas

data = {
    "type": "FeatureCollection",
    "features": [{
            "type": "Feature",
            "geometry": 
                    {
                        "type": "LineString",
                        "coordinates": [
                            [-118, 32], [-119, 33], [-120, 34], [-121, 35], [-122, 36], [-123, 37], [-124, 38]
                        ]
                    },
            "properties": {
                "provider": "MyProvider"
            }
        }]
}

gdf = geopandas.GeoDataFrame.from_features(data)
gdf.explore()
```

When I run this, I was expecting the map tiles to appear as they do in the [introduction tutorial](https://geopandas.org/en/stable/getting_started/introduction.html). However, I see:

![image](https://user-images.githubusercontent.com/98830008/214321348-bb879d73-f6f3-4523-9a61-2ec7558f9abf.png)

What needs to change so this will work?

**Comment by martinfleis** [comment, marked as answer]
GeoPandas doesn't know where on the Earth this is as you haven't specified CRS. If you do this
```
gdf = geopandas.GeoDataFrame.from_features(data, crs="EPSG:4326")
```
and specify EPSG 4326, which means "latitude and longitude", you will see a coast of California underneath.

**Reply by eric-g-97477** [reply]
That works. I had also tried `gdf.set_crs(4326)`, but that didn't work. It seems like it should have. Any idea why it does not?

**Reply by martinfleis** [reply]
It doesn't work in place, you need to assign the result to a variable.
